packages/mint-components/src/components/sqm-leaderboard/sqm-leaderboard.tsx

```diff
@@ -27,12 +26,19 @@ export class Leaderboard {
   /**
    * @uiName Rank Heading
    */
-  @Prop() rankheading?: string;
+  @Prop() rankheading: string;
```

Comment:
Why are we changing this from an optional prop?

Reply:
changed back to optional!